src/winetricks

```diff
@@ -11971,6 +11971,60 @@ __EOF__
 
 #----------------------------------------------------------------
 
+w_metadata office2013pro apps \
+    title="Microsoft Office 2013 Professional" \
+    publisher="Microsoft" \
+    year="2013" \
+    media="cd" \
+    file1="setup.exe" \
+    installed_file1="$W_PROGRAMS_X86_WIN/Microsoft Office/Office15/WINWORD.EXE"
+
+load_office2013pro()
+{
+    if [ "$W_ARCH" = "win64" ]; then
+        w_die "Installation of Office 2013 Pro will not succeed on a 64bit wine prefix"
+    fi
+
+    if [ ! -x "$(which winbindd 2>/dev/null)" ]; then
+        w_warn "winbind is required"
+    fi
+
+    #link from https://www.askvg.com/direct-download-link-microsoft-office-2013-professional-plus-free-trial/
+    if ! test -f "$W_CACHE"/office2013pro/OFFICE15.iso; then
```

Comment:
Don't do that, w_download checks if it exists for you (and this would break if the shasum didn't match)

Reply:
https://github.com/Winetricks/winetricks/pull/939/commits/8b71b66e6f66092167a58c9840ec238d2b1dbf64